scopes/workspace/workspace/pattern.cmd.ts

```diff
@@ -23,7 +23,7 @@ the matching algorithm is done by multimatch (@see https://github.com/sindresorh
     },
     {
       cmd: 'bit pattern "*/ui/**"',
-      description: 'matches components that their namespace starts with "ui/" e.g. "ui/button", "ui/elements/button"',
+      description: 'matches components whose namespace starts with "ui/" e.g. "ui/button", "ui/elements/button"',
```

Comment:
shouldn't it be "start" (without 's') ?

Reply:
it's going on the word 'namespace', which is singular, so the starts has an 's'
don't ask me why, never understood the 'rules' of english grammar...